src/libs/actions/IOU.js

```diff
@@ -61,6 +61,20 @@ Onyx.connect({
     },
 });
 
+let allTransactionDrafts;
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.TRANSACTION_DRAFT,
+    waitForCollectionCallback: true,
+    callback: (val) => {
+        if (!val) {
+            allTransactionDrafts = {};
```

Comment:
Usually, the variable is initialized to `{}` and then this just becomes an early return. That way it's safe (from a code standpoint) to access `allTransactionDrafts` before it's fully initialized.